pandas/core/frame.py

```diff
@@ -6278,6 +6278,54 @@ def swaplevel(self, i: Axis = -2, j: Axis = -1, axis: Axis = 0) -> DataFrame:
         Returns
         -------
         DataFrame
+
+        Examples
+        --------
+        >>> df = pd.DataFrame({'Grade':['A','B','A','C']},
+        ...                   index=[['Final exam', 'Final exam', 'Coursework',
+        ...                              'Coursework'],
+        ...                         ['History','Geography','History','Geography'],
+        ...                         ['January','February','March','April']])
```

Comment:
Could you reformat this according to the `black` formatter (to be consistent with most of the rest of the docs...we should probably run `blackdoc`, but that's for a separate issue/PR)